docs/_docs/02_features/keep-your-remote-state-configuration-dry.md

```diff
@@ -135,6 +137,58 @@ The child `.hcl` file’s `terraform` settings will be merged into the parent fi
 
 Other settings in the child `.hcl` file override the respective settings in the parent.
 
+
+### Using the generate property to generate terraform code for managing remote state
+
+While the default way terragrunt manages remote state is through `terraform init` with `-backend-config`, you can also
+use the `generate` property to configure terragrunt to geneerate a `.tf` file in the terraform working directory with the
+backend configuration.
+
+The `generate` property is an object that accepts two parameters:
+
+- `path`: The path where the generated file should be written. If a relative path, it'll be relative to the Terragrunt
```

Comment:
Does this have a default? E.g., `backend.tf`?

Reply:
No. I think it should be forced to be set given that `backend.tf` feels like a fairly common filename.

Reply:
Roger